### ISSUE TITLE
don't track codechat config file

### ISSUE THREAD

**Issue by StevenClontz** [PreTeXtBook/pretext-codespace#17]
Despite https://github.com/PreTeXtBook/pretext-codespace/blob/main/.gitignore#L98 we are tracking the codechat config file.

**Comment by oscarlevin** [comment]
Since we now create a codechat config file every time pretext is run, should this just never be committed to the template repository?  I think we added it to the repository at some point before we added the line to .gitignore.  But I don't know if there is any other way to have the file be part of the template repository.

**Comment by StevenClontz** [comment]
I don't think we need it here. It will be generated upon any use of the CLI. And it's one less file to overwhelm users.
